moon/controller_round3.py

```diff
@@ -544,16 +562,17 @@ def pol2cart(rho, phi):
                     self.centering = False
 
                 # if seeing basemarker and homebase center is not straight ahead OR if looking past homebase in one of the directions, turn in place to adjust
+                circle_direction = 1 if self.going_around_count % 2 == 0 else -1
```

Comment:
how does the parity of count influence circle direction - please add some explanation comment